README.md

```diff
@@ -34,6 +34,13 @@ A common use-case might be to install local git hooks by setting the `core.hooks
           <custom.configuration>true</custom.configuration> 
         </gitConfig>
       </configuration>
+      <dependencies>
```

Comment:
This is in the wrong section in the readme. This example won't do anything as it is only setting the `core.hooksPath` to point at a directory. You probably wanted this in the second example where you have added `class_path/package/hook.sh`.

Reply:
Have moved to the correct section.